albumentations/augmentations/functional.py

```diff
@@ -294,9 +294,26 @@ def clamping_crop(img, x_min, y_min, x_max, y_max):
 def shift_hsv(img, hue_shift, sat_shift, val_shift):
```

Comment:
I would probably split `shift_hsv` to `shift_hsv_uint ` and `shift_hsv_non_uint`  similar to how it is done in the `adjust_brightness_contrast`